homework.md

```diff
@@ -44,37 +44,39 @@ Using CSS, how would I select every `<h1>` tag on the page?
 
 What is JavaScript?
 
-A programming language used on the internet
+A programming language used on the internet to increase interactivity of sites
 
 ## Question 8
 
 What three problem does Git & Github solve?
 
-When people want to show off code to each other they can put it on Github
+Yes, I expect that is true -When people want to show off code to each other they can put it on Github
+
+Also....To store code, to illustrate the changes and history, to share code and work together.
 
 ## Question 9
 
 What happens when you `fork` a repository?
 
-You delete it
+You copy it and create another version but on your own profile/github website
 
```

Comment:
After that you can make changes on it without affecting the origin .

Reply:
Thank you. Good point!